cloudwatch_metrics_collector/test/cloudwatch_metrics_collector_test.cpp

```diff
@@ -116,7 +116,7 @@ class MetricsCollectorFixture : public ::testing::Test
   }
 
```

Comment:
Test to map the new message to a MetricsObject?

Reply:
Done.